code/model/SiteConfig.php

```diff
@@ -192,48 +205,70 @@ static public function make_site_config() {
 	 * called if a page is set to Inherit, but there is nothing
 	 * to inherit from.
 	 *
-	 * @param mixed $member 
+	 * @param Member $member
 	 * @return boolean
 	 */
-	public function canView($member = null) {
```

Comment:
You remove canView here, but don't later replace it. Is the default from DataObject enough? (You do replace canEdit later)